Cargo.toml

```diff
@@ -1,8 +1,17 @@
 [package]
-name = "crit-cache"
-version = "0.1.0"
-authors = ["Actyx AG <developer@actyx.com", "wngr <oliver@wngr.de>"]
+name = "weight-cache"
+version = "0.1.2"
+authors = ["Actyx AG <developer@actyx.com>", "wngr <oliver@wngr.de>"]
 edition = "2018"
-
+repository = "https://github.com/Actyx/weight-cache"
+license = "MIT OR Apache-2.0"
+categories = ["data-structures"]
+description = "A cache that holds a limited number of key-value-pairs according to a user defined criterion"
+documentation = "https://docs.rs/weight-cache"
 
 [dependencies]
+linked-hash-map = "0.5.4"
```

Comment:
This is the same that LRU uses? I guess it will do for now...

Reply:
It's actually https://crates.io/crates/lru-cache and not https://crates.io/crates/lru. `lru-cache` is more widespread used though. I reviewed the `linked-hash-map` crate and found it to be good.